layouts/explore-layout.vue

```diff
@@ -1,7 +1,11 @@
 <template>
   <div class="min-h-full is-flex is-flex-direction-column is-clipped">
     <Navbar />
-    <ExploreTabsFilterSort />
+    <div class="section pb-0">
+      <div class="container">
+        <ExploreTabsFilterSort />
+      </div>
+    </div>
```

Comment:
we can wrap `main` here, it will allow you not to writte `section.section div.container` two time